testsuite/MDAnalysisTests/topology/test_itp.py

```diff
@@ -490,3 +490,20 @@ def test_missing_elements_no_attribute():
         u = mda.Universe(ITP_atomtypes)
     with pytest.raises(AttributeError):
         _ = u.atoms.elements
+
+
+def test_deprecation_warning():
```

Comment:
```suggestion
def test_elements_deprecation_warning():
```
Just to clarify things a bit for the next developer.